zulipterminal/model.py

```diff
@@ -559,7 +559,8 @@ def get_all_users(self) -> List[Dict[str, Any]]:
     def _stream_info_from_subscriptions(
             subscriptions: List[Dict[str, Any]]
     ) -> Tuple[Dict[int, Any], Set[int], List[List[str]], List[List[str]]]:
-        stream_keys = ('name', 'stream_id', 'color', 'invite_only')
+        stream_keys = ('name', 'stream_id', 'color',
+                       'invite_only', 'description')
```

Comment:
This approach is fine for now - it certainly follows the current logic. It does seem ready for refactoring, though.